src/foam/u2/Element.js

```diff
@@ -38,9 +38,10 @@ foam.ENUM({
   ],
 
   values: [
-    { name: 'CREATE', label: 'Create', modePropertyName: 'createVisibility' },
-    { name: 'VIEW',   label: 'View',   modePropertyName: 'readVisibility'   },
-    { name: 'EDIT',   label: 'Edit',   modePropertyName: 'updateVisibility' }
+    { name: 'CREATE',  label: 'Create',  modePropertyName: 'createVisibility' },
+    { name: 'VIEW',    label: 'View',    modePropertyName: 'readVisibility'   },
+    { name: 'EDIT',    label: 'Edit',    modePropertyName: 'updateVisibility' },
+    { name: 'SUMMARY', label: 'Summary', modePropertyName: 'readVisibility'   }
```

Comment:
Not needed.

Reply:
how we find out where from the approve method is called from without mode?